app/assets/scripts/views/project-browse.js

```diff
@@ -120,6 +120,10 @@ var ProjectBrowse = React.createClass({
         this.props.dispatch(getIndicator(meta.id));
       }
     }
+    const modal = nextState.modal;
+    if (modal !== this.state.modal) {
+      document.documentElement.classList.toggle('disable--page-scroll');
```

Comment:
Nice. This would be cleaner and keep this logic in the same place if you did:
```(javascript)
if (modal) {
  document.documentElement.classList.add('disable--page-scroll');
} else {
  document.documentElement.classList.remove('disable--page-scroll');
}
```

Then you could avoid having the extra call in `closeModal`.